packages/components/src/autocomplete/test/index.tsx

```diff
@@ -21,27 +22,45 @@ describe( 'AutocompleterUI', () => {
 
 			const resetSpy = jest.fn();
 
-			const autocompleter = {
+			const autocompleter: WPCompleter = {
 				name: 'fruit',
+				options: [
+					{ visual: '🍎', name: 'Apple', id: 1 },
+					{ visual: '🍊', name: 'Orange', id: 2 },
+					{ visual: '🍇', name: 'Grapes', id: 3 },
+				],
 				// The prefix that triggers this completer
 				triggerPrefix: '~',
+				getOptionLabel: ( option ) => (
+					<span>
+						<span className="icon">{ option.visual }</span>
+						{ option.name }
+					</span>
+				),
 				// Mock useItems function to return a autocomplete item.
-				useItems: () => {
-					return [
-						[
-							{
-								isDisabled: false,
-								key: 'Apple',
-								value: 'Apple',
-								label: (
-									<span>
-										<span className="icon">🍎</span>
-										{ 'Apple' }
-									</span>
-								),
-							},
-						],
-					];
+				useItems: ( filterValue ) => {
+					const options = autocompleter.options as Array< {
+						visual: 'string';
+						name: 'string';
+						id: number;
+					} >;
+					const keyedOptions = options.map(
+						( optionData, optionIndex ) => ( {
+							key: `${ autocompleter.name }-${ optionIndex }`,
+							value: optionData,
+							label: autocompleter.getOptionLabel( optionData ),
+							keywords: autocompleter.getOptionKeywords
+								? autocompleter.getOptionKeywords( optionData )
+								: [],
+							isDisabled: autocompleter.isOptionDisabled
+								? autocompleter.isOptionDisabled( optionData )
```

Comment:
Will these ternary conditionals ever be `true`?

Reply:
Nope, they weren't actually needed, and have now been removed in 1bd1cfc8d4 as part of the broader fixing up of the unit test.